modern/README.md

```diff
@@ -0,0 +1,194 @@
+These images are mainly focused for Conan Center CI.
+
+#### GCC
+
+| Version                                                                                                   | Arch    |  Status, Life cycle  |
+|-----------------------------------------------------------------------------------------------------------|---------|----------------------|
+| [conanio/gcc5-ubuntu16.04-jenkins: gcc 5](https://hub.docker.com/r/conanio/gcc5-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc6-ubuntu16.04-jenkins: gcc 6](https://hub.docker.com/r/conanio/gcc6-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc7-ubuntu16.04-jenkins: gcc 7](https://hub.docker.com/r/conanio/gcc7-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc8-ubuntu16.04-jenkins: gcc 8](https://hub.docker.com/r/conanio/gcc8-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc9-ubuntu16.04-jenkins: gcc 9](https://hub.docker.com/r/conanio/gcc9-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc10-ubuntu16.04-jenkins: gcc 10](https://hub.docker.com/r/conanio/gcc10-ubuntu16.04-jenkins/)  | x86_64  |  Supported           |
+| [conanio/gcc11-ubuntu16.04-jenkins: gcc 11](https://hub.docker.com/r/conanio/gcc11-ubuntu16.04-jenkins/)  | x86_64  |  Supported           |
+
+
+#### Clang
+
+| Version                                                                                                                  | Arch   |  Status, Life cycle  |
+|--------------------------------------------------------------------------------------------------------------------------|--------|------------|
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 10](https://hub.docker.com/r/conanio/clang10-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 11](https://hub.docker.com/r/conanio/clang11-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 12](https://hub.docker.com/r/conanio/clang12-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+
+
+
+Use the images locally
+======================
+
+You can also use the images locally to build or test packages, this is an example command:
+
+```
+docker run --rm -v /tmp/.conan:/home/conan/.conan conanio/gcc11-ubuntu16.04 bash -c "conan install boost/1.76.0@ --build"
+```
+
+This command is sharing ``/tmp/.conan`` as a shared folder with the conan home, so the Boost package will be built there.
+You can change the directory or execute any other command that works for your needs.
+
+If you are familiarized with Docker compose, also it's possible to start a new container by:
+
+```
+docker-compose run -v /tmp/.conan:/home/conan/.conan gcc11 bash -c "conan install boost/1.74.0@ --build"
+```
+
+
+Build, Test and Deploy
+======================
```

Comment:
I would document first how to build (and test) these images using bare docker (or docker-compose) commands, wdyt? It is what I'd expect when reading about docker.

Then, in another section, we can talk about the `run.py` script, which is related to _batch_ generation of images, some shortcuts to trigger one build after the other (it also uses some py files from CPT, which is unexpected if you just want to build one of these images)

Reply:
That's an excellent addition. I'll add it.

Reply:
Done, I've update README file.